what-is.asciidoc

```diff
@@ -12,24 +12,24 @@ The Ethereum platform enables developers to build powerful decentralized applica
 [[bitcoin_comparison]]
 === Compared to Bitcoin
 
-Many people will come to Ethereum with some prior experience of cryptocurrencies, specifically bitcoin. Ethereum shares many common elements with other open blockchains: a peer-to-peer network connecting participants, a consensus algorithm for synchronization of state (Proof-of-Work), a digital currency (ether) and a global ledger (the blockchain).
+Many people will come to Ethereum with some prior experience of cryptocurrencies, specifically bitcoin. Ethereum shares many common elements with other open blockchains: a peer-to-peer network connecting participants, a consensus algorithm for synchronization of state (proof-of-work system), a digital currency (ether) and a global ledger (the blockchain).
 
 [[blockchain_components]]
 ==== Components of a blockchain
 
 The components of an open, public, blockchain are (usually):
 
-* A peer-to-peer network connecting participants and propagating transactions and blocks, based on a standardized "gossip" protocol.
+* A peer-to-peer network connecting participants and propagating transactions and blocks of verified transactions, based on a standardized "gossip" protocol.
 * A set of consensus rules, implemented in the state machine.
 * Messages, in the form of transactions, representing state transitions.
 * A state machine that processes transactions according to the consensus rules.
 * A distributed database, the blockchain, that records a journal of all the state transitions.
-* A consensus algorithm (e.g. Proof-of-Work) that decentralizes control over the blockchain, by forcing participants to compete and constraining them by the consensus rules.
+* A consensus algorithm (e.g. proof-of-work; or PoW) that decentralizes control over the blockchain, by forcing participants to compete and constraining them by the consensus rules.
```

Comment:
We are keeping "Proof-of-Work" capitalized like that. Otherwise, we would have to change it everywhere. I'll leave that to the O'Reilly editors. Please revert this one change from the PR.